tests_integration/runPrettier.js

```diff
@@ -74,6 +76,9 @@ function runPrettier(dir, args, options) {
   jest.spyOn(require(thirdParty), "getStream").mockImplementation(() => ({
     then: handler => handler(options.input || "")
   }));
+  jest
+    .spyOn(require(thirdParty), "isCI")
+    .mockImplementation(() => process.env.CI);
```

Comment:
Should this be mocked to a specific value so results are consistent?

Reply:
We want to test both CI and non-CI modes and I’m setting env: { CI: true } in a couple of integration tests to make sure the behavior changes.